src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher1.php

```diff
@@ -35,8 +42,8 @@ public function match($pathinfo)
         if (0 === strpos($pathinfo, '/bar')) {
             // bar
             if (preg_match('#^/bar/(?P<foo>[^/]++)$#s', $pathinfo, $matches)) {
-                if (!in_array($this->context->getMethod(), array('GET', 'HEAD'))) {
-                    $allow = array_merge($allow, array('GET', 'HEAD'));
+                if ('GET' !== $isLikeGetMethod) {
+                    $allow[] = 'GET';
```

Comment:
Can we make sure that this is properly tested? If someone else breaks this, it will be a huge bug!